antenna/throttler.py

```diff
@@ -372,6 +407,19 @@ def match_old_buildid(throttler, data):
         condition=lambda throttler, x: x == "ShutDownKill",
         result=(10, CONTINUE, REJECT),
     ),
+    # Accept 25% crash reports from Firefox ESR Windows <= 8.1
+    Rule(
+        rule_name="is_unsupported_firefox_esr_windows_version",
```

Comment:
We should change this rule name to `is_firefox_esr_unsupported_windows` which reads better.